src/EnergyPlus/DataHVACGlobals.cc

```diff
@@ -128,14 +128,15 @@ namespace DataHVACGlobals {
 	Array1D_string const cFurnaceTypes( NumUnitarySystemTypes, { "AirLoopHVAC:Unitary:Furnace:HeatOnly", "AirLoopHVAC:Unitary:Furnace:HeatCool", "AirLoopHVAC:UnitaryHeatOnly", "AirLoopHVAC:UnitaryHeatCool", "AirLoopHVAC:UnitaryHeatPump:AirToAir", "AirLoopHVAC:UnitaryHeatPump:WaterToAir", "AirLoopHVAC:UnitarySystem" } );
 
 	// parameters describing coil types
-	int const NumAllCoilTypes( 29 );
+	int const NumAllCoilTypes( 31 );
 
 	int const CoilDX_CoolingSingleSpeed( 1 );
 	int const CoilDX_HeatingEmpirical( 2 );
 	int const CoilDX_CoolingTwoSpeed( 3 );
 	int const CoilDX_CoolingHXAssisted( 4 );
 	int const CoilDX_CoolingTwoStageWHumControl( 5 );
-	int const CoilDX_HeatPumpWaterHeater( 6 );
+	int const CoilDX_HeatPumpWaterHeaterPumped( 6 );
+	int const CoilDX_HeatPumpWaterHeaterWrapped( 31 );
```

Comment:
Why didn't you just change the numbering 1-n ? It doesn't matter if CoilDX_HeatPumpWaterHeaterWrapped is now 7, but it does matter of a future developer misses the indexing. I guess the compiler will catch it?